### DISCUSSION TITLE
Direct InfluxDB2

### DISCUSSION THREAD

**Discussion by m1k3f15h** [UtilitechAS/amsreader-firmware#419] [Ideas]
Hey Guys,

Is it planed to give the Project a direct InfluxDB2 implementation?
I think, this would be really cool :)
Anyone else would use this?

dears
Mike

**Comment by ArnieO** [comment]
Sounds great! 
I haven't gotten around to using InfluxDB yet, but when I find the time... etc. And an implementation on this project would be a very interesting driver.

**Comment by gskjold** [comment]
I've had this requested a couple of times before and it sounds like an interesting idea. Maybe sometime in the future :)

**Reply by ArnieO** [reply]
> I've had this requested a couple of times before and it sounds like an interesting idea. Maybe sometime in the future :)

I'm sure you would consider a Pull request from @m1k3f15h if he works out something nice? 😉

**Reply by m1k3f15h** [reply]
Good Morning,

I'm sorry if I expressed myself wrongly.
My English is not verry good ;(

i didn't worked out anything. It was only a question if someone other also have a usecase for such a feature :)

dears 
Mike

**Reply by ArnieO** [reply]
No problem! I tried "fishing", as I had a small hope that you had an idea to do an implementation. 😄

**Comment by Noschvie** [comment]
The ESP has limited recourses. Why not using Node-Red to insert the published messages to InfluxDB. Have a look to [IOTstack](https://sensorsiot.github.io/IOTstack)

**Reply by m1k3f15h** [reply]
Hy

Yes i know.
Also IoBroker, FHEM or Telegraf could do this. But i like it when there is no "system" in between :)

**Reply by dbeinder** [reply]
Writing a row into Influx is a really simple operation, just a simple, human-readable HTTP query: https://docs.influxdata.com/influxdb/v2.6/api/#operation/PostWrite

If no one else volunteers, I might implement this. I'll probably only have free time later this year, but I'll check back here before I start working on it.

**Comment by Noschvie** [comment]
Any news? Thanks!

**Reply by gskjold** [reply]
This has yet to reach my priority list I'm afraid, too many other things have kept me busy
